test/cookbooks/minimal/recipes/default.rb

```diff
@@ -29,3 +29,28 @@
   name 'chef test'
   action :disable
 end
+
+windows_task 'create chef test' do
+  name 'chef\\chef test'
+  action :create
+  cwd "C:\\"
+  command 'dir'
+end
+
+windows_task 'disable chef test' do
+  name 'chef\\chef test'
+  action :disable
+end
+
+windows_task 'create chef test' do
+  name '\\chef\\chef test2'
```

Comment:
Why do we need to escape slashes when using a single quoted string?  This isn't intuitive nor how the native commands represent the path of the scheduled tasks.